Nitrox.Launcher/Nitrox.Launcher.csproj

```diff
@@ -43,22 +39,16 @@
     </ItemGroup>
 
     <ItemGroup>
-      <ProjectReference Include="..\NitroxModel\NitroxModel.csproj" />
-      <ProjectReference Include="..\NitroxModel-Subnautica\NitroxModel-Subnautica.csproj" />
-      <ProjectReference Include="..\NitroxServer\NitroxServer.csproj" />
+        <ProjectReference Include="..\NitroxModel\NitroxModel.csproj" />
+        <ProjectReference Include="..\NitroxServer\NitroxServer.csproj" />
     </ItemGroup>
 
-    <!-- Ensure server files are available for launcher. -->
-    <Target Name="BuildServer" AfterTargets="Build">
-        <MSBuild Projects="..\NitroxServer-Subnautica\NitroxServer-Subnautica.csproj" Targets="Build" Properties="Configuration=$(Configuration);OutputPath=$(TargetDir)\Server" />
-    </Target>
```

Comment:
Without server files the launcher can't start the server. Maybe add something like this

```xml
    <Target Name="BuildServer" AfterTargets="Build">
        <Exec Command="dotnet msbuild ..\NitroxServer-Subnautica\NitroxServer-Subnautica.csproj /t:Build /p:Configuration=$(Configuration) /p:OutputPath=$(TargetDir)Server" />
    </Target>
```